db/seeds.rb

```diff
@@ -1 +1,10 @@
 user = User.where(email: "mutualaidtesting@testing.com").first_or_create!(password: "testing123")
+
+SystemTag::DEFAULT_TAGS.each do |tag_name_parent, subtag_name|
+  parent = SystemTag.where(name: tag_name_parent, created_by: "System").first_or_create!
+  if subtag_name.present?
+    SystemTag.where(parent: parent, name: subtag_name, created_by: "System").first_or_create!
+  end
+end
```

Comment:
See my comments about SystemTag nesting.

Reply:
I do see this as being really important bc otherwise the filter list on Share page will get really wonky. This will allow that to be an accordion w subtags.